blake2s.d.ts

```diff
@@ -0,0 +1,22 @@
+export as namespace BLAKE2s;
+
+export type ByteArray = number[] | Uint8Array;
+
+export default class BLAKE2s {
+  constructor(digestLength: number | undefined, keyOrConfig: ByteArray | BLAKE2sConfig);
+  processBlock(length: number): void;
+  update(p: ByteArray, offset?: number, length?: number): this;
+  digest(): Uint8Array;
+  hexDigest(): string;
+  digestLength: number;
+  blockLength: number;
+  keyLength: number;
+  saltLength: number;
+  personalizationLength: number;
```

Comment:
These numbers are `readonly`.

Reply:
Oops, missed that! I'm honestly not sure how to resolve the incompatibility between ES modules and UMD defs, so I've declared them as both `static readonly` members and const fields in the namespace -- that seems least likely to break something for a consumer?